Src/BootCamp.Chapter/Lesson4.cs

```diff
@@ -0,0 +1,110 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Globalization;
+using System.Net.NetworkInformation;
+using System.Text;
+
+namespace BootCamp.Chapter
+{
+    class Lesson4
+    {
+        public static void Demo()
+        {
+            for (int i = 0; i < 2; i++)
```

Comment:
Whatever you have in a loop, split in a method of its' own:
`ProcessPersonInfo()`
or smth like that.